content/shmem_finalize.tex

```diff
@@ -24,7 +24,7 @@
     releases resources used by the \openshmem library. This collective
     operation requires all \acp{PE} to participate in the call. There is an
     implicit global barrier in \FUNC{shmem\_finalize} so that pending
-    communications are completed, and no resources can be released until all
+    communications \newtext{by all threads of the \ac{PE}} are completed, and no resources can be released until all
```

Comment:
@spotluri Yes, we did discuss quite a bit :). As I mentioned on the WG call, this is to emphasize that the User should complete all communications on all threads.